woocommerce/woocommerce
# Replace pink with WooCommerce purple in reports and ratings

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:
Replaced the pink color with WooCommerce purple color on reports page and in the star rating in `WooCommerce recent reviews` widget the dashboard.

### How to test the changes in this Pull Request:

- Go to **WooCommerce > Reports > Orders > Sales by date** and hover on the rows displayed left to the chart (`gross sales in this period`, `net sales in this period` etc. rows). The hover color should be WooCommerce purple.

**Before**
<img width="50%" height="50%" src="https://github.com/woocommerce/woocommerce/assets/33387139/587df65a-c585-4292-a5db-b0010eec7a6a" />

**After**
<img width="50%" height="50%" src="https://github.com/woocommerce/woocommerce/assets/33387139/5434f02c-49ea-4f0c-87e0-0797d05871ab" />


- As a shopper leave a review and rating for any of the products in the shop. In `wp-admin`, go to the product edit page and approve the review. Go to `Dashboard` in your `wp-admin` and scroll down to the `WooCommerce recent reviews` widget. The stars in the rating should have WooCommerce purple color.

**Before**
<img width="50%" height="50%" src="https://github.com/woocommerce/woocommerce/assets/33387139/fd98edb7-6809-482e-92f8-88d6865b7cc7" />

**After**
<img width="50%" height="50%" src="https://github.com/woocommerce/woocommerce/assets/33387139/1b605b01-62d3-47ff-8496-0f819d94c861" />
